pkg/config/loader.go

```diff
@@ -241,6 +252,17 @@ func (l *Loader) mergeAWXCredentials(reference *core.SecretReference) error {
 	return nil
 }
 
+func (l *Loader) mergeAwxJobStatusCheckInterval(intervalStr string) error {
```

Comment:
I think that this code could be included in the caller, instead of having a new function, as it is too small. Not really important, feel free to ignore.